src/service/index.ts

```diff
@@ -648,7 +644,7 @@ class HaloService {
       })
       .filter(Boolean) as string[];
 
-    return [ ...existNames, ...newTags.map((item) => item.data.metadata.name) ];
+    return [...existNames, ...newTags.map((item) => item.data.metadata.name)];
```

Comment:
关于为什么修改这一行
```
      ✖ biome check --staged [FAILED]
↓ Skipped because of errors from tasks.
↓ Skipped because of errors from tasks.
✔ Reverting to original state because of errors...
✔ Cleaning up temporary files...

✖ biome check --staged:
src/service/index.ts format ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━

  × Formatter would have printed the following content:

    647 │ ····return·[·...existNames,·...newTags.map((item)·=>·item.data.metadata.name)·];
        │             -                                                                -

check ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━

  × Some errors were emitted while running checks.
```